tests/system/large/test_remote_function.py

```diff
@@ -2245,3 +2245,130 @@ def test_remote_function_ingress_settings_unsupported(session):
         @session.remote_function(reuse=False, cloud_function_ingress_settings="unknown")
         def square(x: int) -> int:
             return x * x
+
+
+@pytest.mark.parametrize(
+    ("session_creator"),
+    [
+        pytest.param(bigframes.Session, id="session-ctor"),
```

Comment:
I'm not sure what "ctor" means. If it's an abbreviation, could you expand it to a whole word?